frontend/app/framework/utils/date-time.ts

```diff
@@ -186,7 +188,7 @@ export class DateTime {
     }
 
     public toFromNow(): string {
-        return formatDistanceToNow(this.value);
+        return formatDistanceToNow(this.value, { locale: this.getLocale() });
```

Comment:
Please also check the other format methods.

Reply:
Why is this resolved?

Reply:
I was a little bit enthusiastic with pressing that button, before committing.